### ISSUE TITLE
Validating returns_GAFAM.csv file

### ISSUE THREAD

**Issue by jvpagan1** [LemaireJean-Baptiste/eventstudy#6]
Hi. I tried to validate the AAPL returns in the returns_GAFAM.csv file and found differences on the dividend dates. It would appear that the returns are underestimated as if the dividends had been ignored.

From Yahoo Finance
based on
pct_change of
Adj Closing Price
2012-08-09 | 0.005703
2012-11-07 | -0.038263
2013-02-07 | 0.029734
2013-05-09 | -0.008724
2013-08-08 | -0.001991

From returns_GAFAM.csv
date AAPL
2012-08-09 0.001404
2012-11-07 -0.042635
2013-02-07 0.023767
2013-05-09 -0.015242
2013-08-08 -0.008538

Thank you.

**Comment by LemaireJean-Baptiste** [comment]
Hello Jose,
I created the returns_GAFAM.csv file quite a long time ago, using old version of the code. I used this function to retrieve the data.

```python
import pandas as pd
import yf 

def downloadReturn(ticker, start, end):
    data = yf.download(tickers=ticker, start=start, end=end) #Yahoo
    data = data["Close"].pct_change().drop(data.index[0])
    data.columns = ['Stk', 'Idx']
    return data
```

So it appears, we used the same data source (yahoo finance), but for whatever reason, I used the Closing price and not the adjusted close priced as you did. Obviously, choosing adjusted prices or not will change your results, and this choice will depend on what you want to investigate with the event study.

These files are just for testing purpose. I highly recommend to download your own dataset, when doing your own research.

Thanks for your message
